app/en/home/custom-mcp-server-quickstart/page.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: "Custom MCP Server Quickstart"
+description: "Create your custom MCP Server with Arcade MCP"
+---
+
+import { Steps, Tabs, Callout } from "nextra/components";
+import { SignupLink } from "@/app/_components/analytics";
+
+# Custom MCP server Quickstart
+
+<strong>Build AI agents that actually take action.</strong>
+
+Most AI apps are stuck in read-only mode. Arcade gives your AI the power to act — send Gmail, update Notion, message in Slack, and more.
+
+In this quickstart, you'll:
+
+- Install the Arcade MCP Framework
+- Start your MCP server and connect to it from your favorite MCP client
+- Call a simple tool
+- Call a tool that requires a secret
+- Create an Arcade account
+- Call a tool that requires authentication
+
+Ready to start? Follow the Quickstart below.
```

Comment:
Perhaps use the "Outcomes/You will learn/Prerequisites" header like https://docs-git-mateo-tb-quickstart-arcade-ai.vercel.app/en/home/build-tools/create-a-mcp-server